.storybook/preview.js

```diff
@@ -0,0 +1,18 @@
+import 'core-js/stable' // to polyfill ECMAScript features
+import 'regenerator-runtime/runtime' // to use transpiled generator functions
+import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import 'vuetify/dist/vuetify.min.css'
+import Vuelidate from 'vuelidate'
+import Affix from 'vue-affix'
+import Vue2Filters from 'vue2-filters' // needed by SbcFeeSummary
```

Comment:
I think Affix and Vue2Filters are not needed. Please verify and remove accordingly.

Reply:
I copied these from filings-ui, since we are planning to move common components here. For DetailComment we may not need all of it

Reply:
We may or may not move sbc-common-components to this new library. The libraries have slightly different purposes.

If you have another commit, please remove these imports if not needed. If you don't have another commit, we can remove these imports later.

Reply:
removed